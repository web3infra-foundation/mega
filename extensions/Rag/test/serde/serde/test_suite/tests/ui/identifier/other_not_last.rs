use serde_derive::Deserialize;

#[derive(Deserialize)]
#[serde(field_identifier)]
enum F {
    A,
    #[serde(other)]
    Other,
    B,
}

fn main() {}
