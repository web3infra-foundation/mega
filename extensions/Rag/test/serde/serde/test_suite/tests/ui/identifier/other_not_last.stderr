error: #[serde(other)] must be on the last variant
 --> tests/ui/identifier/other_not_last.rs:7:5
  |
7 | /     #[serde(other)]
8 | |     Other,
  | |_________^
